lib/src/main/java/org/team100/lib/config/NotePoseDetector.java

```diff
@@ -1,41 +1,37 @@
 package org.team100.lib.config;
 
-import org.team100.lib.localization.NotePosition24ArrayListener;
-import org.team100.lib.motion.drivetrain.SwerveDriveSubsystem;
-import org.team100.lib.util.CameraAngles;
+import org.team100.lib.copies.SwerveDrivePoseEstimator100;
 
 import edu.wpi.first.math.geometry.Pose2d;
 import edu.wpi.first.math.geometry.Rotation2d;
 import edu.wpi.first.math.geometry.Transform2d;
 import edu.wpi.first.math.geometry.Translation2d;
 
-public class NoteDetector {
-    private final CameraAngles m_cameraAngles;
-    private final SwerveDriveSubsystem m_swerve;
-    private final NotePosition24ArrayListener m_notePosition24ArrayListener;
+public class NotePoseDetector {
+    private final SwerveDrivePoseEstimator100 m_swervePose;
+    private Translation2d m_robotRelativeNoteTranslation = new Translation2d();
```

Comment:
don't use a field here.  it is generally a bad idea for one method to stash something in a field so that some other method picks it up.  just pass it as an argument.